Update install docs (#157)
nextstrain/docs.nextstrain.org: src/install.rst

Installing Nextstrain
=====================

.. contents::
   :local:

.. hint::

    Before installing, we recommend you read about the :doc:`parts of Nextstrain </learn/parts>`.

The following instructions describe how to install the full suite of Nextstrain's software tools, including:

  * Nextstrain CLI, for access to other tools and management of workflows
  * Augur, for bioinformatic analysis of pathogen genomes
  * Auspice, for interactive visualization of pathogen evolution

When completed, you'll be ready to run Nextstrain :term:`workflows <workflow>`.

> 

Reviewer: I'd add a link up here (as well as at the bottom of the page) linking out to "Manage installation" -- the two pages are somewhat separated in the sidebar and I presume many people would come to this page first.

Reviewer: Thanks!

> I don't love it, but the placement of the "Manage installation" page in the sidebar is the best I could think of. Let me know if you have a better suggestion.

I don't think there's a perfect way to order the sidebar. At least, I've never found one.

Optionally, :doc:`configure AWS Batch <cli:aws-batch>` if you'd like to run ``nextstrain build`` on AWS.

> Next, try :doc:`tutorials/running-a-workflow`.

Reviewer 2: I like that splitting out!

            conda deactivate

Next steps
==========

With Nextstrain installed, try :doc:`tutorials/running-a-workflow` next.

Alternate installation methods
==============================

If you want to :doc:`contribute to the development of Nextstrain </guides/contribute/index>` or if you prefer to manage your own custom environment (e.g., a Conda environment, Docker image, environment modules on a cluster, etc.), see the individual installation documentation for :doc:`Nextstrain CLI <cli:installation>`, :doc:`Augur <augur:installation/installation>`, and :doc:`Auspice <auspice:introduction/install>`.

Managing an existing installation
=================================

> See :doc:`this page </guides/manage-installation>` for steps to update, troubleshoot, or uninstall Nextstrain tools.

Reviewer 2: _non-blocking_

I loathe link text like "this page" or "click here". I think just about anything else is better. Smallest change here might be to just use the page title like we do elsewhere:

```suggestion
See :doc:`/guides/manage-installation` for steps to update, troubleshoot, and uninstall Nextstrain.
```

or we could get a bit more diverged:

```suggestion
See our how-to guide with the :doc:`steps to update, troubleshoot, and uninstall Nextstrain </guides/manage-installation>`.
```

Author: Went with the first suggestion in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/675e98db2f8e06feb21ae690edad37249fe33d89..427b38c5be7d9649469743731f01a57d3ed9a4b4).
